libs/dh/admin/feature-user-roles/src/lib/dh-user-roles.component.ts

```diff
@@ -132,8 +144,9 @@ export class DhUserRolesComponent implements OnChanges {
     if (!actor) {
       const actorChanges = {
         id: actorId,
+        atLeastOneRoleIsAssigned: true,
         userRolesToUpdate: { added: [], removed: [] },
-      };
+      } as UpdateUserRolesWithActorId;
```

Comment:
nitpick: doesn't seem to be necessary.

```suggestion
      };
```